extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/DefaultMapboxCarMapGestureHandler.kt

```diff
@@ -0,0 +1,120 @@
+package com.mapbox.maps.extension.androidauto
+
+import androidx.car.app.SurfaceCallback
+import com.mapbox.common.Logger
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.ScreenCoordinate
+import com.mapbox.maps.plugin.animation.camera
+
+/**
+ * This class contains the default map gestures. It automatically captures Android Auto's
+ * [SurfaceCallback] events and applies them to the Mapbox map camera. If you would like customize
+ * the map gestures, use [MapboxCarMap.setGestureDetector].
+ */
+@MapboxExperimental
```

Comment:
IMHO comment is a bit confusing. Should it be smth like
`Handles the gestures received from [SurfaceCallback] and applies them to the `MapboxMap` camera` ?

Reply:
Updated